tests/scripts/common/common_functions.sh

```diff
@@ -457,7 +463,7 @@ function check_server_status() {
 	echo "Server - $SERVER_IP PORT - $PORT"
 
 	#if service does not start within 5 minutes (300s) fail the test
-	timeout 30 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://${SERVER_IP}:${PORT})" != "200" ]]; do sleep 1; done' || false
+	timeout 300 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://${SERVER_IP}:${PORT})" != "200" ]]; do sleep 1; done' || false
```

Comment:
Five mins seems excessive as the job on PR check will be running that long. Why is this needed?

Reply:
This timeout is to check if the HPO service has started or not. If it is not started it will wait for 5 mins before timing out. We can reduce it to 2 mins. Atleast on the github hosted runners, I have seen services taking time to start.

Reply:
Yes, please reduce it to 2 mins, that sounds good